### DISCUSSION TITLE
部署报错哪位大佬帮忙看下

### DISCUSSION THREAD

**Discussion by specialWinter** [labring/sealos#4603] [General]
![image](https://github.com/labring/sealos/assets/25214846/fa8ccf15-de29-4d33-8639-f6cf1418ada2)

**Comment by bxy4543** [comment]
网络问题吧。再来一遍试试

**Reply by specialWinter** [reply]
试了好多次了

**Reply by specialWinter** [reply]
还是不行 快放弃了

**Reply by HAO-WARD** [reply]
是不是虚拟机资源不够？安装sealos私有云官方要求内存至少4G，磁盘至少60G
